plugins/woocommerce/includes/theme-support/class-wc-twenty-twenty-three.php

```diff
@@ -0,0 +1,82 @@
+<?php
+/**
+ * Twenty Twenty THREE support.
+ *
+ * @since   6.0.0
+ * @package WooCommerce\Classes
+ */
+
+use Automattic\Jetpack\Constants;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Twenty_Twenty_Three class.
+ */
+class WC_Twenty_Twenty_Three {
+
+	/**
+	 * Theme init.
+	 */
+	public static function init() {
+
+		// This theme doesn't have a traditional sidebar.
+		remove_action( 'woocommerce_sidebar', 'woocommerce_get_sidebar', 10 );
+
+		// Enqueue theme compatibility styles.
+		add_filter( 'woocommerce_enqueue_styles', array( __CLASS__, 'enqueue_styles' ) );
+
+		// Wrap checkout form elements for styling.
+		add_action( 'woocommerce_checkout_before_order_review_heading', array( __CLASS__, 'before_order_review' ) );
+		add_action( 'woocommerce_checkout_after_order_review', array( __CLASS__, 'after_order_review' ) );
+
+		// Register theme features.
+		add_theme_support( 'wc-product-gallery-zoom' );
+		add_theme_support( 'wc-product-gallery-lightbox' );
+		add_theme_support( 'wc-product-gallery-slider' );
+		add_theme_support(
+			'woocommerce',
+			array(
+				'thumbnail_image_width' => 450,
+				'single_image_width'    => 600,
+			)
+		);
+
+	}
```

Comment:
I wonder if since TT3 is a block theme, and WooCommerce Blocks handles the loading of default block templates if this code should be moved there? 

Presumably any theme that isn't specifically optimized for WooCommerce will need these?